templates/default/hopssite/hs_env.sh.erb

```diff
@@ -18,7 +18,7 @@ export DB_NAME='hops_site'
 export MYSQL_DIR='/srv/hops/mysql-cluster/ndb/scripts/'
 export MSQL_CONNECTOR='mysql-connector-java-5.1.29-bin.jar'
 export HOPS_SITE_WAR="${HOPS_SITE_BASE}/lib/hops-site.war"
-export HOPS_SITE_DOWNLOAD_URL="http://snurran.sics.se/hops/hops-site.war"
+export HOPS_SITE_DOWNLOAD_URL="http://snurran.sics.se/hops/hopssite/hops-site.war"
```

Comment:
Here you shouldn't hardcode snurran.sics.se. Use `node['download_url']` (or something like that) instead.